.releaserc

```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/npm",
+    "@semantic-release/github",
+    "@semantic-release/git"
```

Comment:
I suggest customising the commit message a little bit to match the other repos using Semantic Release.

```suggestion
    [
      "@semantic-release/git",
      {
        "assets": "package.json",
        "message": "chore(release): ${nextRelease.version} [skip ci] \n\n${nextRelease.notes}"
      }
    ]
```

Reply:
I checked and it seems to be like the default https://github.com/semantic-release/git
But will change it for the sake of consistency, good catch!